includes/utilities.inc

```diff
@@ -91,12 +99,17 @@ function islandora_multi_importer_read_filedata($file_path, $numrows = 20, $offs
             // @TODO check if this is not being overriden at line 64
             break;
           }
-          $table[$rowindex] = $datarow[0];
+
+          // If a row begins with a hashtag, it's a comment...skip it
+          // This partially addresses https://github.com/mnylc/islandora_multi_importer/issues/15
+          if ($flat[0] != '#') {
```

Comment:
I have some issues with this. Data could start with a # and if not documented basically people could end having data omitted because of this. Since I added a "reload data from source" button and even select source again, so having multiple sheets some with less data, a simpler workflow? Any other idea on how to make this different? The order of the rows is not important in multi importer, but this sets first as key.  I would love to know about the workflow.  Also, maybe making this a different pull could help talk more about this. Not saying your solution is bad, it is ingenious actually.